typescript/sdk/src/deploy/verify/ContractVerifier.ts

```diff
@@ -67,6 +71,35 @@ export class ContractVerifier {
     );
   }
 
+  private pruneAndMinifyBuildArtifactInput(
```

Comment:
nice wrangling. i would normally suggest a test for a function like this but i think it's low impact if it was wrong and easily detectable if the verification fails